bin/dopublish.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+kob_rel_version=$1
+#kob_archive_download_repo="KOBman_target_repo"
+kob_rel_stage_area=~/tmp
+#kob_namespace="asa1997"
+
+source $HOME/$USR/KOBman/bin/release_var_setup.sh
```

Comment:
remove source command . This will introduce dependency on another file. Only do check if the env variables are set.

Reply:
If any of the variables are not set then exit?

Reply:
Yes. or use the default.  
e.g KOBMAN_NAMESPACE={KOBMAN_NAMESPACE:-hyperledgerkochi}

where hyperledgerkochi is the default.